### DISCUSSION TITLE
#paginatorLeft not working

### DISCUSSION THREAD

**Discussion by geradt** [primefaces/community#75] [PrimeVue] [answered]
I have a PrimeVue datatable component that I'm using in a project and I have enabled pagination in the table. The issue i'm having is, up until now I've been able to have the pagination buttons pulled to the right under the table. That's still working just fine. The issue I'm having is on the left side. I was adding in custom text that says "Showing 1 to 50 of 2877 records". That was working up until I updated the primeVue package recently. Did something change with the #paginatorLeft slot? Was it renamed or do I have to accomplish this differently now? Thanks in advance!

Code:
```
<template #paginatorLeft v-if="totalRecords">
     <div class="p-l-sm">Showing {{ from }} to {{ to }} of {{ totalRecords }} records</div>
</template>
```

**Comment by FlipWarthog** [comment, marked as answer]
I think you want `#paginatorstart`. See: https://primevue.org/datatable/#paginator_template and https://primevue.org/datatable/#api.datatable.slots.paginatorstart

**Reply by geradt** [reply]
Thank you! I just found that in the docs as well. looks like the slot name changed with a recent release. Thanks FlipWarthog!
